pkgs/by-name/vu/vuetorrent/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vuetorrent";
+  version = "2.18.0";
+
+  src = fetchzip {
+    url = "https://github.com/VueTorrent/VueTorrent/releases/download/v${version}/vuetorrent.zip";
+    hash = "sha256-Z+N1RgcF67R6hWEfmfBls1+YLWkhEJQuOVqXXJCyptE=";
+  };
+
+  installPhase = ''
```

Comment:
needs `runHook preInstall` and `runHook postInstall`

Reply:
Sorry, that wasn't clear. Yes, the first line of the `installPhase` should be `runHook preInstall` and the last line should be `runHook postInstall`